src/Components/Components/src/Routing/Router.cs

```diff
@@ -8,6 +8,9 @@
 using System.Collections.ObjectModel;
 using System.Linq;
 using System.Reflection;
+using System.Reflection.Metadata.Ecma335;
+using System.Security.Cryptography;
+using System.Threading;
 using System.Threading.Tasks;
```

Comment:
Are these namespaces really needed?

Reply:
System.Threading is for the TaskCancellationSource. Don't think the others are.